tests/attr/test_shapley.py

```diff
@@ -279,6 +285,19 @@ def _shapley_test_assert(
             assertTensorTuplesAlmostEqual(
                 self, attributions, expected_attr, delta=delta, mode="max"
             )
+            if test_true_shapley:
+                shapley_val = ShapleyValues(model)
+                attributions = shapley_val.attribute(
+                    test_input,
+                    target=target,
+                    feature_mask=feature_mask,
+                    additional_forward_args=additional_input,
+                    baselines=baselines,
+                    perturbations_per_eval=batch_size,
+                )
+                assertTensorTuplesAlmostEqual(
+                    self, attributions, expected_attr, mode="max", delta=0.001
```

Comment:
could we use the same delta as above ?

Reply:
We could, but that delta is almost always substantially larger since sampling has pretty wide errors, this is much closer, so I fixed a smaller value here.